test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerTests.cs

```diff
@@ -258,11 +258,17 @@ public static TheoryData<CreateTokenTheoryData> CreateJWEUsingSecurityTokenDescr
                         TestId = "TokenDescriptorClaimsNull",
                         Payload = new JObject()
                         {
-                            { JwtRegisteredClaimNames.Aud, Default.Audience }
+                            { JwtRegisteredClaimNames.Aud, Default.Audience },
+                            { JwtRegisteredClaimNames.Exp, EpochTime.GetIntDate(Default.Expires).ToString() },
+                            { JwtRegisteredClaimNames.Iat, EpochTime.GetIntDate(Default.NotBefore).ToString() },
```

Comment:
nit: can we have a Default.IssuedAt?

Reply:
I didn't notice this earlier, but we do have a property called Default.IssueInstant. I've switched to using this value for the 'iat' claim.